addons/mail/static/src/chatter/web_portal/chatter.js

```diff
@@ -47,10 +42,7 @@ export class Chatter extends Component {
         Dropdown,
         Thread,
         Composer,
-        Activity,
```

Comment:
This removal should be done on the same commit as updating the template, otherwise it doesn't really make sense to reference a component that is no longer imported.